sechub-cli/src/mercedes-benz.com/sechub/cli/job.go

```diff
@@ -147,8 +147,11 @@ func printSecHubJobSummaryAndFailOnTrafficLight(context *Context) {
 		}
 	case "GREEN":
 		fmt.Println("  GREEN - no severe security vulnerabilities identified")
+	case "OFF":
+		fmt.Println("  SCAN FAILED - detection of security vulnerabilities failed on server or product")
+		os.Exit(ExitCodeFailed)
 	case "":
```

Comment:
As discussed: I would suggest to use the same mechanism as for "" also for "OFF". So we have the same error message and user shall inspect the status messages from server side